AK/HashMap.h

```diff
@@ -40,6 +40,11 @@ class HashMap {
             set(item.key, item.value);
     }
 
+    HashMap(HashMap const&) = default; // FIXME: Not OOM-safe! Use clone() instead.
+    HashMap(HashMap&& other) noexcept = default;
+    HashMap& operator=(HashMap const& other) = default; // FIXME: Not OOM-safe! Use clone() instead.
+    HashMap& operator=(HashMap&& other) noexcept = default;
```

Comment:
How do these comments "reveal" that we copy-construct and copy-assign HashMaps very often?

Reply:
You're right, "reveal" is a bit too strong. How about "underlines"?